src/main/java/com/jcabi/aspects/aj/MethodCacher.java

```diff
@@ -72,34 +70,19 @@
      * @checkstyle LineLength (2 lines)
      */
     private final transient ConcurrentMap<MethodCacher.Key, MethodCacher.Tunnel> tunnels =
-        new ConcurrentHashMap<MethodCacher.Key, MethodCacher.Tunnel>(0);
+        new ConcurrentHashMap<Key, Tunnel>(0);
 
     /**
-     * Service that cleans cache.
+     * Save the keys which need update.
      */
-    private final transient ScheduledExecutorService cleaner =
-        Executors.newSingleThreadScheduledExecutor(
-            new NamedThreads(
-                "cacheable",
-                "automated cleaning of expired @Cacheable values"
-            )
-        );
+    private final transient BlockingQueue<Key> updatekeys =
```

Comment:
This variable was used in this class, and if we want to get rid of it from here, we must add a new method to the class 'UpdateMethodCacher'.